### DISCUSSION TITLE
Cannot launch the glsp-examples

### DISCUSSION THREAD

**Discussion by dinkoivanov-gepard** [eclipse-glsp/glsp#1103] [Q&A]
Hi,

I'm trying to run the node-json-vscode example by following the readme https://github.com/eclipse-glsp/glsp-examples/blob/master/README.md.
I manage to build the project with yarn, but when I try to run it, the server starts ok:

```
4:57:13 PM info -  [SocketServerLauncher] The GLSP server is ready to accept new client requests on port: 5007 
[GLSP-Server]:Startup completed
```

But the client reports an error:

```
[JsonrpcGLSPClient] Connection to server is erroring. Shutting down server. (1) [Error: connect ECONNREFUSED 127.0.0.1:5007
	at TCPConnectWrap.afterConnect [as oncomplete] (node:ne…]
[JsonrpcGLSPClient] Connection to server is erroring. Shutting down server. (1) [Error: connect ECONNREFUSED 127.0.0.1:5007
	at TCPConnectWrap.afterConnect [as oncomplete] (node:ne…]
rejected promise not handled within 1 second: Error: Connection got disposed.
stack trace: Error: Connection got disposed.
	at Object.dispose (/Users/dinkoi/projects/vmware/wdl-editor-2/glsp-client/node_modules/vscode-jsonrpc/lib/main.js:876:25)
	at /Users/dinkoi/projects/vmware/wdl-editor-2/glsp-client/node_modules/@eclipse-glsp/protocol/lib/jsonrpc/base-jsonrpc-glsp-client.js:73:24
	at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Please help,
Thanks,
Dinko

**Comment by rhumbertgz** [comment]
@dinkoivanov-gepard I have the same issue on macOS and other also on windows. Using a linux machine or a container should work. I'm using the dev container of the node-json-theia and it works.

**Comment by planger** [comment]
Thanks for raising this issue. [Others](https://github.com/eclipse-glsp/glsp/discussions/1090) have observed this recently as well and a bug has been raised: https://github.com/eclipse-glsp/glsp/issues/1100

We didn't get to investigate this yet further, but certainly plan to look into at soon, as it seems to affect many people. If you have any further insights or possibly a fix already, please feel free to share your findings! Thank you!
